examples/demo-rollup/tests/bank_integration_tests.rs

```diff
@@ -0,0 +1,81 @@
+mod test_helpers;
+use std::net::SocketAddr;
+
+use borsh::BorshSerialize;
+use demo_stf::app::DefaultPrivateKey;
+use demo_stf::runtime::RuntimeCall;
+use jsonrpsee::core::client::{Subscription, SubscriptionClientT};
+use jsonrpsee::rpc_params;
+use jsonrpsee::ws_client::WsClientBuilder;
+use sov_modules_api::default_context::DefaultContext;
+use sov_modules_api::transaction::Transaction;
+use sov_modules_api::{PrivateKey, Spec};
+use sov_sequencer::utils::SimpleClient;
+use test_helpers::start_rollup;
+
+const TOKEN_SALT: u64 = 0;
+const TOKEN_NAME: &'static str = "test_token";
+
+async fn send_test_create_token_tx(rpc_address: SocketAddr) -> Result<(), anyhow::Error> {
+    let key = DefaultPrivateKey::generate();
+    let user_address: <DefaultContext as Spec>::Address = key.to_address();
+
+    let token_address = sov_bank::get_token_address::<DefaultContext>(
+        TOKEN_NAME,
+        user_address.as_ref(),
+        TOKEN_SALT,
+    );
+
+    let msg = RuntimeCall::bank(sov_bank::CallMessage::<DefaultContext>::CreateToken {
+        salt: TOKEN_SALT,
+        token_name: TOKEN_NAME.to_string(),
+        initial_balance: 1000,
+        minter_address: user_address,
+        authorized_minters: vec![],
+    });
+    let tx = Transaction::<DefaultContext>::new_signed_tx(&key, msg.try_to_vec().unwrap(), 0);
+
+    let port = rpc_address.port();
+    let http_client = SimpleClient::new(&format!("http://localhost:{}", port));
+    let ws_client = WsClientBuilder::default()
```

Comment:
This is great! What about moving it inside  `SimpleClient` and adding a relevant method there?